docs/usage/LandscaperDeployments.md

```diff
@@ -26,6 +26,9 @@ spec:
       - helm
       - manifest
       - container
+  oidcConfig: # optional
+    clientID: <some client ID>
+    issuerURL: <OIDC token issuer URL>
```

Comment:
userClaim and groupClaim is missing I think

Reply:
very good point